src/event/views/EventPage.vue

```diff
@@ -33,16 +33,12 @@
             @update="switchCalendar()"
           />
           <div class="buttons">
-            <AppButton
-              class="btn"
-              text="Save Response"
-              @update="handleSave()"
-            />
-            <AppButton
-              class="btn"
-              text="Copy Event Link"
-              @update="copyLink()"
-            />
+            <AppButton variant="secondary" class="btn" @update="handleSave()"
+              >Save Response</AppButton
+            >
+            <AppButton variant="secondary" class="btn" @update="copyLink()"
+              >Copy Event Link</AppButton
+            >
```

Comment:
Did the linter do this? Kind of weird to have the closing tag on a different line for the </AppButton>

Reply:
Yeah but it might be more from Prettier. This happens a lot in my code. I kinda just got used to it..